CMakeLists.txt

```diff
@@ -944,6 +945,7 @@ if(${PLAYER_BUILD_EXECUTABLE} AND ${PLAYER_TARGET_PLATFORM} STREQUAL "SDL2")
 	endif()
 elseif(${PLAYER_TARGET_PLATFORM} MATCHES "^(psvita|3ds|switch|wii)$")
 	add_executable(easyrpg-player "src/main.cpp")
+	install(TARGETS easyrpg-player RUNTIME DESTINATION . COMPONENT debug)
```

Comment:
just a note as this confused me: This ``RUNTIME DESTINATION .`` looks strange but relative paths are rooted at ``CMAKE_INSTALL_PREFIX``.
(So is correct)

Reply:
Yep, so it is possible to even "install" to a mounted SD card.